packages/react-components/src/components/Badge/Badge.tsx

```diff
@@ -3,22 +3,33 @@ import * as React from 'react';
 import styles from './Badge.module.scss';
 import cx from 'clsx';
 
+const baseClass = 'badge';
```

Comment:
```suggestion
const baseClass = 'lc_badge';
```
?
I don't know if we should still apply `lc_` prefix, but I saw a comment in the other PR that mentioned that.

Reply:
No. We don't use the `lc-` prefix in class names.

Reply:
If we found any in other components, we should fix that.

Reply:
@AlithAnar AFAIR we're avoiding prefixes in order to avoid conflicts with the legacy version of the Design System. I remember also there should be a possibility to apply a custom prefix globally, but I'm not sure if it was implemented.